packages/ui-components/src/components/advanced-date-select-dropdown/advanced-date-select-dropdown.tsx

```diff
@@ -70,6 +64,8 @@ export class KvAdvancedDateSelectDropdown implements IAdvancedDateSelectDropdown
 
 	componentWillLoad() {
 		this.handleSelectedTimeChange(this.selectedTime);
+
+		this.internalSelectedTimezone = this.selectedTimezone;
```

Comment:
You should create a handle to assign the `selectedTimezone` to the `internalSelectedTimezone` when the first changes.